client/Brocfile.js

```diff
@@ -1,11 +1,23 @@
-/* global require, module */
-
+/* global require, module, process */
+var env = process.env.EMBER_ENV;
```

Comment:
yes that does look like a good plugin.  We are currently adding the [GA property id to ENV](https://github.com/hairbrain/gallery13/blob/fix_56/client/config/environment.js#L7) also.  We are only using the environment variable when loading the config object to know what environment is being run.  

I wonder if the current approach gives us more flexibility to control how and when the events are fired compared to ember-cli-google-analytics.  Pageview event tracking is typically the baseline analytics that gets setup and then adding event tracking is typically the next thing that is asked for.  It seems to me that the current approach will give us more flexibility long term w/o having to submit PR's to ember-cli-google-analytics to add this logic.  I may be wrong, I haven't looked into the details of that plugin beyond a quick read of their README...